src/components/AnalyticCard/AnalyticCard.js

```diff
@@ -102,24 +104,26 @@ class AnalyticCard extends Component {
                   }}
                 />
 
-                <Text><b>{firstText}</b></Text>
+                <Text>
+                  <b>{firstText}</b>
+                </Text>
                 <Text>{secondText}</Text>
               </div>
             </CardContent>
           </CardSection>
-          {this.props.helpTextId
-            ? <CardDrawers
-                onClick={this.handleDrawerClick}
-                active={this.state.activeDrawer}
-                drawers={[
-                  {
-                    id: 'help',
-                    name: formatMessage({ id: 'container.drawer.help' }),
-                    content: <FormattedMarkdown text={this.props.helpTextId} />
-                  }
-                ]}
-              />
-            : null}
+          {this.props.helpTextId ? (
```

Comment:
`{this.props.helpTextId && <CardDrawers...` would be better

Reply:
These are all formatting changes, so trying not to change anything unrelated to my updates.

Reply:
Sounds good!